src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -112,6 +112,20 @@ const TwoWindingsTransformerCreationDialog = ({
     const [ratioCellIndexError, setRatioCellIndexError] = useState(undefined);
     const [phaseCellIndexError, setPhaseCellIndexError] = useState(undefined);
 
+    const computePhaseHighTapPosition = () => {
```

Comment:
Why define nearly twice the same function ?
You may well define one with a parameter and give it once phaseTapRows and ratioTapRows the other.
Especially as valueLessThanOrEqualTo is not expected to be a function, no ?

Reply:
There's no need, I implemented it your way. I thought it would cause issues at validation to pass a value computed at initialization instead of a reference but after testing it it works well